metals/src/main/scala/scala/meta/metals/providers/HoverProvider.scala

```diff
@@ -59,10 +59,14 @@ object HoverProvider extends LazyLogging {
         info.symbol.stripSuffix(".").parse[Term].get.asInstanceOf[Term.Ref]
       Pkg(ref, Nil)
     } else if (info.kind.isTrait) {
+      // TODO: include type parameters
       Defn.Trait(mods, Type.Name(info.name), Nil, EmptyCtor, EmptyTemplate)
     } else if (info.kind.isClass) {
+      // TODO: include type parameters and primary constructor
```

Comment:
I worry about TODO vs tickets.  Consider creating issues for these and referencing the issue in the TODO comment.